docs/monitoring/monitoring-metricbeat.asciidoc

```diff
@@ -0,0 +1,127 @@
+[role="xpack"]
+[[monitoring-metricbeat]]
+=== Monitoring {kib} with {metricbeat}
+
+beta[] In 6.4 and later, you can use {metricbeat} to collect data about {kib} 
+and ship it to the monitoring cluster, rather than routing it through the 
+production cluster as described in <<monitoring-kibana>>. 
+
+image::monitoring/images/metricbeat.png[Example monitoring architecture]
```

Comment:
Love that you added this!

Reply:
Great, I'll do the same for the Elasticsearch page.